app/src/ui/multi-commit-operation/choose-branch/merge-choose-branch-dialog.tsx

```diff
@@ -127,12 +125,17 @@ export class MergeChooseBranchDialog extends BaseChooseBranchDialog {
 
     if (this.state.selectedBranch !== branch) {
       this.commitCount = 0
+      return
```

Comment:
Same reason as above -> if this has changed, no need to continue to update the preview.